### PR TITLE
Fix tags type

### PR DESCRIPTION
**Pull request by doom-weaver**
### Notes
Tags type only allows a single item, literally keyed "string", in the `{ "string": <value of tag>: string}` type definition.

Fix this so that it allows any number of string typed keys with string typed values

### CODE REVIEW

**Comment by adambollen** [comment]
Looks like `package-lock.json` has a ton of changes - is that expected?

**Comment by doom-weaver** [comment]
> Looks like `package-lock.json` has a ton of changes - is that expected?

All i did is run npm install. 

I think what happened was this branch in the package.lock refers to a different version of the driver:

https://github.com/fauna/faunadb-js/blob/fauna_logs/package-lock.json#L3

compare:

https://github.com/fauna/faunadb-js/blob/3468f6e7a86d77284e54eff8cebea09d0d531b91/package-lock.json#L3

**Comment by adambollen** [comment]
> I think what happened was this branch in the package.lock refers to a different version of the driver:
> 
> https://github.com/fauna/faunadb-js/blob/fauna_logs/package-lock.json#L3
> 
> compare:
> 
> https://github.com/fauna/faunadb-js/blob/3468f6e7a86d77284e54eff8cebea09d0d531b91/package-lock.json#L3

Oh, that makes sense.  We'll need to bump that to `-beta.2` before releasing also.

File: src/types/Client.d.ts [hunk 1]
```diff
@@ -37,7 +37,7 @@ export interface QueryOptions
   > {
   signal?: AbortSignal
   traceparent?: string
-  tags?: { string: string }
+  tags?: { [key: string]: string }
```
Comment (adambollen) [inline src/types/Client.d.ts:40]: I'm a little confused; when I was running queries with multiple tags, they were showing up correctly in the logs.  Shouldn't I expect that not to work?

Comment (doom-weaver) [reply src/types/Client.d.ts:40]: did you use typescript or raw javascript?

If you use types it would refuse to compile.

Comment (adambollen) [reply src/types/Client.d.ts:40]: Oh right, that's the ticket.  Thanks.

Comment (doom-weaver) [reply src/types/Client.d.ts:40]: > Oh, that makes sense. We'll need to bump that to `-beta.2` before releasing also.

Sure I can do that now.

**Review by adambollen — APPROVED** [review]

**Pull request merged**